server/build.gradle.kts

```diff
@@ -0,0 +1,144 @@
+plugins {
+    kotlin("jvm")
+    id("maven-publish")
+    id("application")
+    id("com.github.jk1.tcdeps") version "1.2"
+    id("com.jaredsburrows.license") version "0.8.42"
+}
+
+val projectVersion = project.property("projectVersion").toString()
+val kotlinVersion = project.property("kotlinVersion").toString()
+val exposedVersion = project.property("exposedVersion").toString()
+val lsp4jVersion = project.property("lsp4jVersion").toString()
+val javaVersion = project.property("javaVersion").toString()
+val debugPort = 8000
+val debugArgs = "-agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n,quiet=y"
+
+version = projectVersion
+
+val mainClassName = "org.javacs.kt.MainKt"
+val applicationName = "kotlin-language-server"
+
+application {
+    mainClassName = mainClassName
```

Comment:
`mainClassName` seems to be shadowed here, so it isn't set to the variable defined above. This causes the language server to crash on startup on my machine. If I try to run the language server from the command line (`./server/build/install/server/bin/kotlin-language-server`) it does not seem to set up the java-command correctly, as it prints the usage helper. This seems to be due to mainClass not being set correctly, and therefore no main class name being used in the Java command. (the resulting script simply runs java without any arguments causing the usage-message).

Got everything working if used another name for the variable in line 19 🙂

Reply:
Ok, thank you for helping with out on the testing front @themkat! I've renamed it on line 19 and renamed the references under the `application` block and `debugRun` + `debugStartScripts` tasks and squashed the change into the original commit.